presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -114,8 +114,8 @@
 
 import static com.google.common.base.Strings.isNullOrEmpty;
 import static com.google.common.collect.Comparators.lexicographical;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
 import static io.prestosql.plugin.hive.HiveErrorCode.HIVE_METASTORE_ERROR;
-import static io.prestosql.plugin.hive.HiveErrorCode.HIVE_PARTITION_DROPPED_DURING_QUERY;
```

Comment:
We should consistently use `PartitionNotFoundException` or `HIVE_PARTITION_DROPPED_DURING_QUERY`. Now it's mixed.

Reply:
We should continue to use this error code. `PartitionNotFoundException` is only for drop. The usage in `SemiTransactionalHiveMetastore#finishInsertIntoExistingPartition` seems wrong and should be `HIVE_PARTITION_DROPPED_DURING_QUERY`.